src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -0,0 +1,683 @@
+/** Histogram Visualization. */
+loadScript("https://d3js.org/d3.v4.min.js");
+loadStyle("https://fontlibrary.org/face/dejavu-sans-mono");
```

Comment:
please load the things only when visualization is used! Not during the start of GUI, as this slows down things drastically.